.github/workflows/screenshot.yml

```diff
@@ -35,8 +35,9 @@ jobs:
       
       - name: (Screenshot) Screenshot Pull Request Changes
         run: |
-          ./packages/cli/bin/bmc.js screenshot --output .artifacts/visual-snapshots
-
+          ./packages/cli/bin/bmc.js serve &
+          docker run --rm -v $PWD:$PWD ghcr.io/linz/basemaps-screenshot/cli@1.0.0 --url http://localhost:5000 --output $PWD/.artifacts/visual-snapshots
```

Comment:
I assume this won't be working before we merge this and release? Shall we relocate first then switching when deprecating the old one?

Reply:
I have already moved the command into its new spot https://github.com/linz/basemaps-screenshot it just hasnt released the container yet.

It also looks like using `&` to keep the server running just works :cool:

Reply:
I see, Thanks. I think we need to release the screenshot container first before merging this.